src/scss/index.scss

```diff
@@ -38,8 +38,8 @@ $header-height--narrow            : 56px;
 $header-height                    : 76px;
 $header-height--narrow--multiLang : $header-height--narrow + $languageSwitcher-height;
 $header-bg                        : $white !default;
-$header-separator                 : 4px;
-$header-border-color              : $red !default;
+$header-separator                 : 1px;
+$header-border-color              : $grey--lightest !default;
```

Comment:
Will this change the header separator on all pages?  If so, will need the designs to be updated and approved here so that they stay up-to-date > https://app.zeplin.io/project/5925866ae4bbe61fb7fd7988

Reply:
Also, will need to update designs like these too.

https://app.zeplin.io/project/58e7554e0352f8ef2215f131/screen/58b41f3494c5115391929235

Not sure if this would be better to make as a modifier to the current header, as will need to go through a lot of the global designs and update this so that we have references to this new style.

Reply:
>modifier to the current header
As far as i'm aware design have all agreed this if going to be the new header for all - so a modifier seems like additional styles that will be unused - maybe a major version bump?
I've passed the above Zeplins to Piet to update :)

Reply:
COol – it looks like these have been updated now in Zeplin so happy for this to go in as the default 👍